### ISSUE TITLE
c_DMGCPU::Tick Segfault

### ISSUE THREAD

**Issue by Quaker762** [wornwinter/DMGe#20]
Why the hell would *if(running)* be causing a segfault in the debugger?
http://imgur.com/Tbg7bp7

**Comment by wornwinter** [comment]
I haven't managed to get this issue. Were there any steps you took before it happened?

**Comment by Quaker762** [comment]
Nope, just ran it with the debugger. Might be the laptop I'm using though, considering flash and IE constantly crash.

**Comment by wornwinter** [comment]
Might be. I think that gdb and Windows just don't play nice together lol. I'll maybe start using Linux for debugging.
